addons/udes_security/models/res_archiving_restrictions.py

```diff
@@ -0,0 +1,82 @@
+from odoo import fields, models, _, SUPERUSER_ID
+from odoo.exceptions import ValidationError
+
+
+class ResArchivingRestrictions(models.Model):
+    """
+    Configurable access to _only_ record archiving & unarchiving
+
+    If an Archiving Restriction is created for a model, the group_ids set will determine
+    which groups have the facility to archive and unarchive records on the model.
+
+    This includes backend writes to the `active` field, as well as hiding
+    the Archive/Unarchive action from the model, and attempting to remove active
+    fields & smart buttons from form views.
+    """
+
+    _name = "res.archiving.restrictions"
+    _description = "Record Archiving Restrictions"
+    _sql_constraints = [
+        (
+            "model_id_uniq",
+            "unique(model_id)",
+            _(
+                "An Archiving Restriction already exists for this model. Please edit the existing one"
```

Comment:
Super super super nit, feel free to resolve but I think the message should be set as one of the following:

`An Archiving Restriction already exists for this model, please edit the existing one`

or

`An Archiving Restriction already exists for this model. Please edit the existing one.`

or just:

`An Archiving Restriction already exists for this model`